src/ast/variables/LocalVariable.ts

```diff
@@ -167,7 +167,7 @@ export default class LocalVariable extends Variable {
 				);
 			}
 			case INTERACTION_ASSIGNED: {
-				if (this.included) return true;
+				if (this.included && !context.ignore.this) return true;
```

Comment:
@lukastaegert I'm not sure why these tests are failing on CI but succeeding on my local machine, and It seems that all failing tests are related to `sourcemap`, Do you have any idea? By the way, If I remove this change, the tests succeed.

Reply:
No idea, they also appear to be green for me locally. But besides that, I think running this logic for every variable means important side effects will be lost. Look at this example:

https://deploy-preview-4842--rollupjs.netlify.app/repl/?pr=4842&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQW51bGwlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIyY29kZSUyMiUzQSUyMmxldCUyMHglMjAlM0QlMjAxJTNCJTVDbiU1Q25jbGFzcyUyMEZvbyUyMCU3QiU1Q24lMjAlMjAlMjAlMjBjb25zdHJ1Y3RvcigpJTIwJTdCJTVDbiUyMCUyMCUyMCUyMCUyMCUyMHglMjAlM0QlMjAyJTNCJTVDbiUyMCUyMCUyMCUyMCUyMCUyMHRoaXMub3B0aW9uMSUyMCUzRCUyMDEyMyUzQiU1Q24lMjAlMjAlMjAlMjAlN0QlNUNuJTdEJTVDbiU1Q25uZXclMjBGb28oKSUzQiU1Q24lNUNuJTJGJTJGJTIwc2hvdWxkJTIwbG9nJTIwJTVDJTIyMiU1QyUyMiU1Q25jb25zb2xlLmxvZyh4KSUzQiUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTJDJTIybmFtZSUyMiUzQSUyMm1haW4uanMlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyb3V0cHV0JTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyZXMlMjIlN0QlMkMlMjJ0cmVlc2hha2UlMjIlM0F0cnVlJTdEJTdE

Reply:
Oh, I didn't think about that.